src/support/z_context.erl

```diff
@@ -416,10 +417,12 @@ abs_url(Url, Context) ->
 
 
 %% @doc Fetch the pid of the database worker pool for this site
+-spec db_pool(#context{}) -> atom().
 db_pool(#context{db={Pool, _Driver}}) ->
     Pool.
 
 %% @doc Fetch the database driver module for this site
+-spec db_driver(#context{}) -> atom().
```

Comment:
The spec in line 425 uses a record, please define a type for the record and use that instead.